docs/source/backends/create_your_backend.rst

```diff
@@ -93,3 +93,41 @@ The previous example can be used with the following class definition
             converter = []
             parser = BaseParser
             renderer = BaseRenderer
+
+Once you have your python package configured with the correct entry points
+you should have a directory tree that looks like this.
+
+.. code-block:: bash
+
+    $ tree example_backend
+    .
+    ├── example_backend
+    │   └── __init__.py
+    └── setup.py
+
+And now you can install your package using `pip install example_backend`.
+
+If you install both `netjsonconfig` and `example_backend` you can use your backend
```

Comment:
shouldn't netjsonconfig be installed anyway because it's a dependency?
Maybe we should update the `setup.py` example to have `netjsonconfig` in `install_requires`, what do you think?

Reply:
Until yesterday the plugin feature was not released so it didn't cross my mind. I'll add the dependency to the setup file example